src/screens/DetailDomain/elements/DetailDomainScreenHeader.js

```diff
@@ -83,9 +83,9 @@ const styles = StyleSheet.create({
   headerDomainInfoContainer: {flex: 1, marginRight: 8},
   headerDomainDateContainer: {flexDirection: 'row', alignItems: 'center', flexShrink: 1},
   wrapperText: {
-    backgroundColor: COLORS.white,
+    backgroundColor: 'white',
     borderRadius: 8,
-    borderColor: COLORS.holyTosca,
+    borderColor: '#00ADB5',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color values can make the code less maintainable. It's better to use a centralized color configuration, which allows for easier updates in the future. Consider reverting back to using the `COLORS` object for color definitions.
```diff
-    backgroundColor: 'white',
-    borderColor: '#00ADB5',
+    backgroundColor: COLORS.white,
+    borderColor: COLORS.holyTosca,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->